src/rofi_rbw/entry.py

```diff
@@ -7,3 +8,10 @@ class Entry:
     name: str
     folder: Optional[str] = ""
     username: Optional[str] = ""
+
+    def sha1(self) -> str:
+        m = sha1()
```

Comment:
Any reason you chose `sha1` over others?

Reply:
I guess it's the fastest which is not deprecated